### ISSUE TITLE
Error on tagging the image

### ISSUE THREAD

**Issue by mpepping** [pangzineng/Github-Action-One-Click-Docker#1]
Despite not having set `DOCKER_REGISTRY_URL` in my workflow, it seems that still the IF-condition is true (in https://github.com/pangzineng/Github-Action-One-Click-Docker/blob/master/entrypoint.sh#L39L46), and docker tag fails with a leading '/' in the tag name:

My workflow:

```
workflow "Build and push to Docker Hub" {
  on = "push"
  resolves = [
    "build-and-push-to-docker-hub",
  ]
}

action "build-and-push-to-docker-hub" {
  uses = "pangzineng/Github-Action-One-Click-Docker@master"
  secrets = [
    "DOCKER_USERNAME",
    "DOCKER_PASSWORD",
  ]
  env = {
    DOCKER_IMAGE_NAME = "pdk-centos"
  }
}
```

The error:

```
 ---> 64df3313ac89
Successfully built 64df3313ac89
Successfully tagged pdk-centos:latest
Error parsing reference: "/[FILTERED]/pdk-centos:master" is not a valid repository/tag: invalid reference format
Error parsing reference: "/[FILTERED]/pdk-centos:660086d" is not a valid repository/tag: invalid reference format
Error parsing reference: "/[FILTERED]/pdk-centos:latest" is not a valid repository/tag: invalid reference format
invalid reference format
invalid reference format
invalid reference format

### FAILED build-and-push-to-docker-hub 20:58:16Z (1m43.428s)
```

**Comment by pangzineng** [comment]
Thanks for reporting the bug. 
It was my mistake for not testing the Docker Hub case.
I fixed the bug and rolled up the version to `v1.1.1`, it should be fine now.

**Comment by mpepping** [comment]
All good! Thanks for the fast update.
